tools/build.js

```diff
@@ -436,6 +436,9 @@ function publish() {
                     stdio: 'inherit',
                     cwd: path.join(distPath, package),
                 });
+            } catch (e) {
```

Comment:
This was the only functional change in this file, yes?

Reply:
Yes.